### PR TITLE
Collection card redesign

### LINKED ISSUE #4497: Collection card
image

- profile picture of the collection
- banner, we dont have the implementation yet so we can use last sold image or scaled profile picture instead. 
- name, if long - cropped wit 3 dots
- divider 
- 3 properties - set to space-between so they take up the space. 

hover effect - same as nft cards 
hover on the collection name - text color to k-hovergrey, in dark mode - from white to shade -> collection hover card should pop up.

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [x] Refactoring

## Context

- [x] Closes #4497
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

image

image

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1075,6 +1075,12 @@
     "join": "Join Our Community",
     "subscribeLabel": "Subscribe"
   },
+  "collectionCard": {
+    "volume": "Volume",
+    "highestSale": "Highest Sale",
```
Comment (roiLeo) [inline locales/en.json:1080]: already exist, it would be duplicated

File: locales/en.json [hunk 2]
```diff
@@ -1075,6 +1075,12 @@
...
+    "items": "Items"
```
Comment (roiLeo) [inline locales/en.json:1081]: use `totalItems`

**Review by roiLeo — COMMENTED** [review]
Wow, I love this new design

### Some Notes
- could you use rem instead of px for font-size?
- can we update `CollectionList` file outside rmrk folder?
- I am not really convinced of the BEM methodology, some css rules can be writter using bulma helpers
- no gutter on mobile
- image is missing a border in the `collection-detail__header` but I don't think it's doable

File: locales/en.json (hunk 2 above)
Comment (Jarsen136) [reply locales/en.json:1080]: the translation word for `totalItems` is `Σ Items`, different from that on the figma. I'm not sure if use it.

**Comment by Jarsen136** [comment]
> ### Some Notes
> * could you use rem instead of px for font-size?
> * can we update `CollectionList` file outside rmrk folder?
> * I am not really convinced of the BEM methodology, some css rules can be writter using bulma helpers
> * no gutter on mobile

✅ Updated

> * image is missing a border in the `collection-detail__header` but I don't think it's doable
I think so : ) Otherwise, I could make three lines to wrap it.

**Comment by prachi00** [comment]
@Jarsen136 something scary 👻  is happening when hovering, I am not sure if its just me, or happening for you too
https://www.loom.com/share/26cd784b38c7410ebbeac80823501e3f

**Comment by roiLeo** [comment]
> @Jarsen136 something scary 👻 is happening when hovering, I am not sure if its just me, or happening for you too 

it doesn't happen to me, what OS & Browser are you using?

File: locales/en.json (hunk 2 above)
Comment (roiLeo) [reply locales/en.json:1080]: meh.. it's the same, I ask you to use the keys already present as some have already been translated no need to duplicate them

**Review by roiLeo — APPROVED** [review]
### Next step: (for collection card)
- histoire component (like #4511)
- refactor `CollectionList` component + Vue3 way
- choose OOCSS over BEM 
- clear legacy code
- change collection schema "type" to be chainless
- remove unused `isLoading` prop
- skelton view onload?
- extend collection sorting options

**Comment by Jarsen136** [comment]
> > @Jarsen136 something scary 👻 is happening when hovering, I am not sure if its just me, or happening for you too
> 
> it doesn't happen to me, what OS & Browser are you using?

It doesn't happen to me, too.

File: locales/en.json (hunk 2 above)
Comment (Jarsen136) [reply locales/en.json:1080]: I have removed `highestSale` key in the latest commit. I guess refreshing the page could see.

**Comment by yangwao** [comment]
@exezbcz 👀

**Comment by yangwao** [comment]
noticed a broken arrow on landing on this PR 
image

`main` branch looks like this 
image

**Comment by Jarsen136** [comment]
> noticed a broken arrow on landing on this PR 

It has been fixed, after merging `main` branch into this PR 😄

**Comment by prachi00** [comment]
> > @Jarsen136 something scary 👻 is happening when hovering, I am not sure if its just me, or happening for you too
> 
> it doesn't happen to me, what OS & Browser are you using?

MacOs, google chrome

**Comment by Jarsen136** [comment]
> seems like the feedback from @exezbcz for this PR? cc @Jarsen136
> 
> [#4506 (comment)](https://github.com/kodadot/nft-gallery/pull/4506#issuecomment-1360241384)

Thanks for hinting me. Got it.

**Comment by exezbcz** [comment]
😅😂😂😂

**Comment by Jarsen136** [comment]
>background of the card should be k-darkmode, not black
is it also possible to make the stroke around the frame as in design?

✅ Updated

image

**Comment by yangwao** [comment]
I would be up to merge it 😅

@exezbcz

**Comment by exezbcz** [comment]
its looking good ✅

**Comment by yangwao** [comment]
pay 70 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $70 @ 24.35 USD/KSM ~ 2.875 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x61cdcbe689692f9f41783f8cfe997147c57d5530f4df30b63e03e7048277d82c](https://kusama.subscan.io/extrinsic/0x61cdcbe689692f9f41783f8cfe997147c57d5530f4df30b63e03e7048277d82c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**